test/openAPI/features/data_create.feature

```diff
@@ -27,7 +27,7 @@ Feature: API endpoint that allows users to create a new record in the database.
 
     Examples: Valid data
     | ID         |Firstname | LastName | BirthCertificateID
-    | EE34534123 | Zofia    | Don      | RR-5465679229
-    | EE38778473 | Elsa     | Smith    | RR-1234537999
+    | ${ID_2} | Zofia    | Don      | RR-5465679229
+    | ${ID_3} | Elsa     | Smith    | RR-1234537999
     | RR87483474 | Nick     | Evans    | RR-7898797999
     | PP84848484 | Thomas   | Johnson  | RR-8534567781
```

Comment:
If we're replacing the data that is put in here I believe that we shouldn't keep hardcoded data.

Reply:
True, done